packages/core/src/types.ts

```diff
@@ -1285,3 +1285,14 @@ export type ActorRefFrom<
   : never;
 
 export type AnyInterpreter = Interpreter<any, any, any, any>;
+
+export type InterpreterFrom<
```

Comment:
Could you also reexport this from the root file? While accessing `/types` is currently possible it is not recommended and will probably break in the future.

Reply:
as @davidkpiano has pointed out to me, this is already handled: https://github.com/davidkpiano/xstate/blob/14cdef612df29b9e067a34ca96b0750e729c9aff/packages/core/src/index.ts#L77